scripts/build/build.sh

```diff
@@ -24,6 +24,9 @@ header "Checking Pre-Reqs"
 
 $REPOROOT/scripts/test/check-prereqs.sh
 
+# Increases the file descriptors limit for this bash. It prevents an issue we were hitting during restore
+ulimit -n 512
```

Comment:
We would still be making an assumption about the current limit, because while it varies from OS, it truly varies by machine. I may have changed mine for instance and break our assumption. Is it worth investing in maybe getting the current value and comparing before increasing?! I set it to 512 because it was one where both of us have seen the build succeed with it.